index.yaml

```diff
@@ -1,43 +1,46 @@
-    version: 1.0.2
-  - apiVersion: v2
-    appVersion: 0.0.1
-    created: "2024-07-15T12:41:59.105775839Z"
-    description: 'Helm chart to deploy the Orchestrator solution suite on OpenShift,
-      including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator,
-      Knative Eventing and Knative Serving. '
-    digest: 364d523b8a36277b3f0062b1a396a3703da55292de24e6c698fda3a6644eb511
-    icon: https://raw.githubusercontent.com/parodos-dev/parodos-dev.github.io/main/assets/images/WO_black.svg
-    name: orchestrator
-    type: application
-    urls:
-    - https://github.com/parodos-dev/orchestrator-helm-chart/releases/download/orchestrator-1.0.1/orchestrator-1.0.1.tgz
-    version: 1.0.1
+    - apiVersion: v2
+      appVersion: 0.0.1
+      created: "2024-08-01T05:13:56.557304562Z"
+      description:
+        "Helm chart to deploy the Orchestrator solution suite on OpenShift,
+        including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator,
+        Knative Eventing and Knative Serving. "
+      digest: 2b9fa38ff05b5814f337f3f1294c4722003903e2d1068ded9f220f7b87f40b21
+      icon: https://raw.githubusercontent.com/rhdhorchestrator/rhdhorchestrator.github.io/main/assets/images/WO_black.svg
+      name: orchestrator
+      type: application
+      urls:
+        - https://github.com/rhdhorchestrator/orchestrator-helm-chart/releases/download/orchestrator-1.0.3/orchestrator-1.0.3.tgz
+      version: 1.0.3
+    - apiVersion: v2
+      appVersion: 0.0.1
+      created: "2024-07-25T06:42:19.995728421Z"
+      description:
+        "Helm chart to deploy the Orchestrator solution suite on OpenShift,
+        including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator,
+        Knative Eventing and Knative Serving. "
+      digest: 44c7156c7cefd657ae5767487f45889fbb4ae1c00dfba8232ae692d5ab03ab43
+      icon: https://raw.githubusercontent.com/rhdhorchestrator/rhdhorchestrator.github.io/main/assets/images/WO_black.svg
+      name: orchestrator
+      type: application
+      urls:
+        - https://github.com/rhdhorchestrator/orchestrator-helm-chart/releases/download/orchestrator-1.0.2/orchestrator-1.0.2.tgz
+      version: 1.0.2
+    - apiVersion: v2
+      appVersion: 0.0.1
+      created: "2024-07-15T12:41:59.105775839Z"
+      description:
+        "Helm chart to deploy the Orchestrator solution suite on OpenShift,
+        including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator,
+        Knative Eventing and Knative Serving. "
+      digest: 364d523b8a36277b3f0062b1a396a3703da55292de24e6c698fda3a6644eb511
+      icon: https://raw.githubusercontent.com/rhdhorchestrator/rhdhorchestrator.github.io/main/assets/images/WO_black.svg
+      name: orchestrator
+      type: application
+      urls:
+        - https://github.com/rhdhorchestrator/orchestrator-helm-chart/releases/download/orchestrator-1.0.1/orchestrator-1.0.1.tgz
+      version: 1.0.1
```

Comment:
can you please double check if the indentation check is needed in this PR?

Reply:
Oops, stayed there by mistake. I'll revert it :)